scripts/elastic_search/loaders/aggregate_loader.py

```diff
@@ -1,68 +1,95 @@
 from mapping import ESMapping
 from loaders import *
+from annotators import *
 from files import *
 from mods import *
+import gc
+import time
 
 import os
 
 class AggregateLoader:
 
-    def load_from_mods(self):
-        mods = [RGD(), MGI(), ZFIN(), SGD(), WormBase(), FlyBase(), Human()]
+    def __init__(self):
+        self.go_dataset = {}
+        self.so_dataset = {}
+        self.batch_size = 5000 # Set size of gene batches created from JSON file.
+        self.test_set = 'false' # Limit dataset to 100 gene entries from each MOD.
+        self.threads = 2 # Set number of CPU threads used for indexing.
+    
+    def establish_index(self):
+        print "ES_HOST: " + os.environ['ES_HOST']
+        print "ES_INDEX: " + os.environ['ES_INDEX']
+        print "ES_AWS: " + os.environ['ES_AWS']
+        self.es = ESMapping(os.environ['ES_HOST'], os.environ['ES_INDEX'], os.environ['ES_AWS'])
+        self.es.start_index()
+
+    def load_from_files(self):
+        print "Loading data from saved files"
+        self.go_dataset = PickleFile("tmp/go_bkp.pickle").load()
+        self.so_dataset = PickleFile("tmp/so_bkp.pickle").load()
 
+    def load_annotations(self):
         print "Loading GO Data"
-        go_data = GoLoader().get_data()
-        print "Loading OMIM Data"
-        omim_data = OMIMLoader().get_data()
-        print "Loading SO Data"
-        so_loader = SoLoader()
-
-        self.genes = {}
-        print "Gathering genes from Each Mod"
-        for mod in mods:
-            self.genes.update(mod.load_genes())
+        self.go_dataset = GoLoader().get_data()
+        print "Loading SO Data" 
+        self.so_dataset = SoLoader().get_data()
+
+
+    def load_from_mods(self, pickle, index):
+        mods = [RGD(), MGI(), ZFIN(), SGD(), WormBase(), FlyBase(), Human()]
+        #mods = [FlyBase()]
```

Comment:
I get why this is here, so you can do a single mod while developing, but it would be cool at some point (not for this PR) to make this something that can be controlled with an argument.